staging/src/k8s.io/kubectl/pkg/cmd/apply/apply_test.go

```diff
@@ -2263,7 +2266,12 @@ func TestLoadObjects(t *testing.T) {
 func TestApplySetParentManagement(t *testing.T) {
 	// TODO: replace with cmdtesting.InitTestErrorHandler() when the feature is fully implemented
 	cmdutil.BehaviorOnFatal(func(s string, i int) {
-		if s != "error: ApplySet-based pruning is not yet implemented" {
+		switch s {
+		case "error: ApplySet-based pruning is not yet implemented":
```

Comment:
This error is never expected anymore as of this PR, and I'm guessing the error below is only expected because the test setup needs to be updated to handle the delete. So as of this PR we should be ready to do the TODO above (and similar in other tests). I can do it in a follow up if you prefer.

Reply:
Good call - fixed (and the other copy of this code block)